src/components/core/text/text.tsx

```diff
@@ -31,11 +31,15 @@ export const color_array = [
 export const align_array = ['left', 'center', 'right'] as const;
 
 type TTextProps = {
```

Comment:
```suggestion
type StyledTextProps = Omit<Stitches.ComponentProps<typeof StyledText>, 'dark'>;

interface ITextProps extends StyledTextProps {
    as?: React.ElementType;
}
```
what do you think about this approach @hamid-deriv ? 
this way we'll have all the props from the `StyledText` at once.

Reply:
Changed.